src/views/Documentation/PageMeta.jsx

```diff
@@ -112,26 +147,38 @@ export default class PageMeta extends Component {
               [classes.pageNavigationWithoutPrevious]: !hasPreviousPage,
             })}>
             {hasPreviousPage && (
-              <Button
+              <PageTransition
                 onClick={this.handlePreviousPage}
-                color="secondary"
                 variant="extended"
                 aria-label="Previous Page"
                 className={classes.navigationButton}>
-                <ChevronLeftIcon />
-                {pageInfo.prev.title}
-              </Button>
+                <ArrowLeftIcon color="#000" className={classes.leftIcon} />
+                <div className={classes.rowFlex}>
+                  <div className={classes.leftButtonText}>
+                    <span className={classes.smallText}>Prev</span>
+                    <span className={classes.largeText}>
+                      {pageInfo.prev.title}
+                    </span>
```

Comment:
Instead of using `span`s, let's use the [`Typography`](https://material-ui.com/api/typography/) component.